### PR TITLE
Add CloudTrail support.

### PR DESCRIPTION
**Pull request by arimbun**
Add support for the following checks:

* `have_global_service_events_included`
* `be_multi_region_trail`
* `have_log_file_validation_enabled`

### CODE REVIEW

**Comment by k1LoW** [comment]
Wow!!!! Great work!! 🎉 Thank you!!!

File: lib/awspec/generator/doc/type.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def self.generate_doc
         doc = ERB.new(header, nil, '-').result(binding)
 
         types.map do |type|
+          Awspec::Generator::Doc::Type::Cloudtrail.new
```
Comment (k1LoW) [inline lib/awspec/generator/doc/type.rb]: This line is not necessary. 👍

**Comment by k1LoW** [comment]
@arimbun Please check https://github.com/k1LoW/awspec/pull/162#discussion_r70937921

Thank you for your contribution!

File: lib/awspec/generator/doc/type.rb (hunk 1 above)
Comment (arimbun) [inline lib/awspec/generator/doc/type.rb]: Ah yes, was debugging! Will remove.

**Comment by k1LoW** [comment]
LGTM! Thank you!!!!!

**Comment by k1LoW** [comment]
Released as v0.46.0.

**Pull request merged**